components/organisms/SideMenu/@types/SideMenu.ts

```diff
@@ -0,0 +1,10 @@
+import { IButtonProps } from "components/molecules/Button/@types/Button";
+import { IHorizontalLinksWithIconsProps } from "components/molecules/HorizontalLinksWithIcons/@types/HorizontalLinksWithIcons";
+import { INavProps } from "components/molecules/Nav/@types/Nav";
+
+export interface ISideMenuProps {
+  icons?: IHorizontalLinksWithIconsProps["icons"];
```

Comment:
```suggestion
  icons: IHorizontalLinksWithIconsProps["icons"];
```

`icons` are always passed down to `SideMenu`